packages/ndla-ui/src/Resource/ListResource.tsx

```diff
@@ -85,14 +97,37 @@ export interface ListResourceProps {
 
 const ListResource = ({ link, title, tags, resourceImage, topics, description, actionMenu }: ListResourceProps) => {
   const showDescription = description !== undefined;
+
+  function CheckTagsLength(tags: string[]) {
+    if (tags.length > 3) {
+      return (
+        <>
+          <TagList tags={tags.slice(0, 3)} />{' '}
+          <MenuButton
+            hideMenuIcon={true}
+            menuItems={tags.slice(3, tags.length).map((tag) => {
+              return {
+                text: '#' + tag,
+                onClick: () => {},
+              };
+            })}>
+            <TagCounterWrapper>+ {tags?.length - 3}</TagCounterWrapper>
+          </MenuButton>
+        </>
+      );
+    } else {
+      return <TagList tags={tags} />;
+    }
+  }
+
   return (
     <ResourceWrapper to={link}>
       <StyledImage alt={resourceImage.alt} src={resourceImage.src} imageSize={showDescription ? 'normal' : 'compact'} />
       <ResourceInfoWrapper>
         <Row>
           <ResourceTitle>{title}</ResourceTitle>
-          <TagList tags={tags} />
-          {actionMenu}
+          {tags && CheckTagsLength(tags)}
+          <MenuButton size="small" />
```

Comment:
Denne `MenuButton` vil ikke fungere fordi den ikke mottar noen props for tilvalg i menyen,

Enten må actionMenu brukes som før, eller så må ListResource ta i mot nødvendige props for å lage en fullt fungerende MenuButton (da burde samme metode også implementeres i BlockResource)

Reply:
endrer tilbake til actionmenu